webcompat/static/js/lib/wizard/utils.js

```diff
@@ -97,3 +97,16 @@ export const getDataURIFromPreview = (bgImage) => {
 
   return match[1];
 };
+
+/* Trim wyciwyg://N/ from URL, if found.
```

Comment:
In theory this was fixed last year by https://bugzilla.mozilla.org/show_bug.cgi?id=1489308, so we could possibly remove `wyciwyg` handling entirely. I think it's fine for you to leave in, perhaps we can just file a follow-up contributor bug.